SwapnilPatle.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Swapnil Patle",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Front End Developer.Learner,Youtuber",
+  "avatar": "https://github.com/settings/profile",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SwapnilPatle01",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Follow me on YouTube",
+      "url": "https://www.youtube.com/channel/UCn0F3wdlhYyS3G5mO8li8Jg",
+      "icon": "FaYouTube"
+    },
+     {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/swappy___01/",
+      "icon": "FaInstagram"
+    } ,
+     {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.facebook.com/mdswapnil.patle",
+      "icon": "FaLinkedin"
+    },
+     {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/mdswapnil.patle",
+      "icon": "FaInstagram"
+    },
+     {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/settings/profile",
```

Comment:
This is not your GitHub URL. Your GitHub URL goes something like this `https://github.com/<YOUR-USERNAME>`. Eg `https://github.com/AbbeyIT`